configuration/configuration.go

```diff
@@ -160,35 +160,11 @@ type Configuration struct {
 	// registry events are dispatched.
 	Notifications Notifications `yaml:"notifications,omitempty"`
 
-	// Redis configures the redis pool available to the registry webapp.
-	Redis struct {
-		// Addr specifies the the redis instance available to the application.
-		Addr string `yaml:"addr,omitempty"`
-
-		// Password string to use when making a connection.
-		Password string `yaml:"password,omitempty"`
-
-		// DB specifies the database to connect to on the redis instance.
-		DB int `yaml:"db,omitempty"`
-
-		DialTimeout  time.Duration `yaml:"dialtimeout,omitempty"`  // timeout for connect
-		ReadTimeout  time.Duration `yaml:"readtimeout,omitempty"`  // timeout for reads of data
-		WriteTimeout time.Duration `yaml:"writetimeout,omitempty"` // timeout for writes of data
-
-		// Pool configures the behavior of the redis connection pool.
-		Pool struct {
-			// MaxIdle sets the maximum number of idle connections.
-			MaxIdle int `yaml:"maxidle,omitempty"`
-
-			// MaxActive sets the maximum number of connections that should be
-			// opened before blocking a connection request.
-			MaxActive int `yaml:"maxactive,omitempty"`
+	RedisDBs struct {
+		LayerInfoCache Redis `yaml:"layerinfocache,omitempty"`
+	}
```

Comment:
This would be a breaking change for existing configs. Not sure if that's a concern for the maintainers, but if it is we could retain the existing field and add `RedisDBs` as a new config field, then assert during initialization that at most one of the two is filled in. E.g.:

```go
Redis Redis `yaml:"redis,omitempty"`
RedisDBs struct {
    LayerInfoCache Redis `yaml:"layerinfocache,omitempty"`
} `yaml:"redisdbs,omitempty"`
```